tests/test_templates.py

```diff
@@ -0,0 +1,157 @@
+import logging
+
+import numpy as np
+
+from pinttestdata import datadir
+from pint.templates import lcfitters, lcprimitives, lctemplate
+
+
+def gauss(x, x0, s):
```

Comment:
Should this be using the bin-averaged gaussian rather than the sampled gaussian? This is available from scipy.stats.norm.cdf. The sampled version gets erratic when the width gets small compared to a sample.

Reply:
It would make the derivatives pretty expensive.

Reply:
Moreover, in the internal caching code, I believe it does evaluate the integrated model + derivatives, albeit using something like a Simpson's rule, iirc.  But this test is just of the point evaluation of the profile, anyway.